redbot/cogs/alias/alias.py

```diff
@@ -182,7 +182,9 @@ def get_extra_args_from_alias(
         """
         Manage global aliases.
         """
-        if ctx.invoked_subcommand is None or isinstance(ctx.invoked_subcommand, commands.Group):
+        if ctx.invoked_subcommand is None or isinstance(
+            ctx.invoked_subcommand, discord.ext.commands.Group
+        ):
```

Comment:
Instead of using `isinstance`, why not simply do an equality check with the group object itself? This technique works with multi-level subgroups, and I'd like to see it used more :smiley: